docs/contracts.rst

```diff
@@ -85,9 +108,13 @@ To run this example, you will need to install a few extra features:
 
     # Display the new greeting value
     print('Updated contract greeting: {}'.format(
-        greeter.functions.greet().call()
+    greeter.functions.greet().call()
     ))
 
+    # When issuing a lot of reads, try this more concise reader:
```

Comment:
Lines 114-116 should be removed. We are no longer encouraging the use of ConciseContract in v5